### PR TITLE
Transform any element into a clickable link

### PR DESCRIPTION
**Pull request by lgavillet**

### CODE REVIEW

File: app/assets/javascripts/mtl/data_href.coffee [hunk 1]
```diff
@@ -0,0 +1,6 @@
+MTL.onReady ->
```
Comment (lwe) [inline app/assets/javascripts/mtl/data_href.coffee]: not required, we can simply bind directly with `$(document).on ...`

**Comment by lwe** [comment]
Also please update `CHANGES.md`, thanks

**Comment by lgavillet** [comment]
Append to the existing list or add a new version?

**Comment by lwe** [comment]
sorry, one more thing, in keeping consistency with `select.coffee` this should be named `href.coffee`

**Comment by lwe** [comment]
Simply append, as we are not yet at a stable verison

**Comment by lwe** [comment]
Sorry, **prepend**

**Comment by lwe** [comment]
Cool, thanks 👍

**Pull request merged**